### ISSUE TITLE
Failure to receive invites from Riot interface

### ISSUE THREAD

**Issue by lrq3000** [vector-im/element-web#11957]
New user here, on Windows 10 x64 (on both computers), by typing exactly the user matrix id and inviting them, they got no notification on their side from the application, although the discussion was opened (but it was with a "ghost").

The only way I have found to make the inviting work was by clicking on the link sent by email (which by the way is not hinted at anywhere in the app, I did not expect to get an email). Then, it worked correctly. Weirdly, this opened a new discussion, unrelated to the one that the application opened itself when I invited the contact, but this might be because I clicked the link of invitation (since it was not working correctly, we both tried to invite each other, so my initial discussion I opened before the invitation was accepted was me -> other, and when I clicked the link I accepted the discussion other -> me, but theoretically this should be exactly the same since it's bidirectional!).

In summary: I invited someone by the user matrix id (could not find by username), this opened a discussion (me -> other), there was no notification from the app but an email was received, when I clicked the link this opened a new discussion (other -> me) and I deleted the old one (me -> other).

It's not a problem of id collision, I'm pretty sure both of our IDs are unique. We are both using the latest stable release, v1.5.7.

**Comment by turt2live** [comment]
This sounds a bit like you expected to enter a user ID but instead entered an email, which is why it's a bit awful. Matrix IDs are in the form of `@user:domain.com` while emails would be `user@domain.com` - a very subtle but easy mistake to make.

Since January we've clarified some of the text in the invite dialogs to be more clear, and we have a number of issues regarding email invites. I don't think we need this issue, though I am sorry that the invite didn't go well :(

**Comment by lrq3000** [comment]
Unfortunately no mistake on my part, i specifically tried to use the matrix id as explained in the issue.

However this was with an old version of riot, now it may be different with element and i have yet to try again since i do not add new contacts nowadays.

**Comment by lrq3000** [comment]
Ps although i might also have tried to type the email in after failure with the matrix id, which may explain why i got an email, but honestly i don't remember nor understand why i got that email as i wrote in the issue.

The only thing i know for sure is that i typed the matrix id, this created a new discussion thread, but my contact did not see any of my messages nor saw the thread appear on their end, it was like a ghost thread.
